web/src/components/layout/Panel.vue

```diff
@@ -1,23 +1,63 @@
 <template>
   <div class="rounded-md w-full shadow overflow-hidden bg-gray-300 dark:bg-dark-gray-700">
-    <div v-if="title" class="font-bold text-gray-200 bg-gray-400 dark:bg-dark-gray-800 px-4 py-2">{{ title }}</div>
-    <div class="w-full p-4 bg-white dark:bg-dark-gray-700 text-color">
-      <slot />
+    <component
+      :is="collapsable ? 'button' : 'div'"
+      v-if="title"
+      type="button"
+      class="flex w-full font-bold gap-2 text-gray-200 bg-gray-400 dark:bg-dark-gray-800 px-4 py-2"
+      @click="collapsed = !collapsed"
```

Comment:
```suggestion
      @click="isCollapsable && (collapsed = !collapsed)"
```

Maybe preventing the click in case of the div would be a good idea as well 🤔